### ISSUE TITLE
Train Models

### ISSUE THREAD

**Issue by ARUNAGIRINATHAN-K** [ARUNAGIRINATHAN-K/ARUNAGIRINATHAN-K#2]
# Linear Regression
from sklearn.linear_model import LinearRegression
lr = LinearRegression().fit(X_train, y_train)

# Random Forest
from sklearn.ensemble import RandomForestRegressor
rf = RandomForestRegressor().fit(X_train, y_train)

# SVM
from sklearn.svm import SVR
svm = SVR().fit(X_train, y_train)

# XGBoost
import xgboost as xgb
xgb_model = xgb.XGBRegressor().fit(X_train, y_train)

# LSTM
from keras.models import Sequential
from keras.layers import LSTM, Dense

X_lstm = X.values.reshape((X.shape[0], 1, X.shape[1]))
X_train_lstm, X_test_lstm = X_lstm[:len(X_train)], X_lstm[len(X_train):]

lstm = Sequential([
    LSTM(50, return_sequences=True, input_shape=(1, X.shape[1])),
    LSTM(50),
    Dense(1)
])
lstm.compile(loss='mean_squared_error', optimizer='adam')
lstm.fit(X_train_lstm, y_train, epochs=10, batch_size=32)
